### DISCUSSION TITLE
CALIBRATION FRSKY HORUS 10S

### DISCUSSION THREAD

**Discussion by djrobbs65** [EdgeTX/edgetx#4029] [Q&A]
Hi everyone, I switched from opentx where everything worked well, to edgetx, and the problem I'm having is on the calibration on the left stick, which does not perform all the movements of the circle, while the right stick does, (tried both with the latest firmware, 2.9, and with 2.8), while with opentx, everything worked perfectly, I also tried to reflash several times, but nothing! I don't know what could have happened. I obviously followed the official procedure of the site:
https://buddy.edgetx.org/#/flash?version=v2.9.0, and everything ran perfectly without any errors.
I hope you experts can help me solve it, otherwise I will have to go back to opentx.
I look forward to your kind technical support.
Thank you

**Comment by raphaelcoeffic** [comment]
@djrobbs65 Compared to OpenTx, the calibration procedure is basically the same. It might be helpful to check the raw values. You can see them in the radio hardware settings (see [here](https://edgetx.gitbook.io/edgetx-user-manual/edgetx-user-manual/user-manual-for-color-screen-radios/radio-settings/hardware#debug)). The first tab show the calibrated values, while the third tab shows raw values as we get them from the stick (prior to applying calibration).

**Comment by djrobbs65** [comment]
Ciao e grazie per la risposta, ora mi trovo in una situazione che lo stick
fisico sinistro quando sta al centro, non corrisponde al centro. Del
cerchio del display…. Ho provato a fare. La calibrazione. Piano piano varie
volte.. Ma risulta fuori di un tacca

Il lun 11 set 2023, 11:02 Raphael Coeffic ***@***.***> ha
scritto:

**Reply by raphaelcoeffic** [reply]
Sorry, I don't understand any italian :-D

**Comment by djrobbs65** [comment]
Hello and thanks for the reply, now I find myself in a situation where the
left physical stick when it is in the center does not correspond to the
center.  Of the display circle….  I tried to do.  Calibration.  Slowly
several times.. But it comes out a notch

Il lun 11 set 2023, 19:12 Raphael Coeffic ***@***.***> ha
scritto:

**Reply by raphaelcoeffic** [reply]
During calibration, there is a step where it says you should have the sticks (and pots) in the middle. The software remembers exactly this position and uses it as the center. If you don't have it exactly where it should be at this point, it will stay this way after calibration. Don't bother about the display at this stage. You should see the correct position in the next step once you start moving to the sides and up&down.

**Comment by djrobbs65** [comment]
perfect, you were very kind!!  I did not think.  to start putting the stick
in the center... because I saw that on the display it told me off center
and therefore.  I thought I was doing the wrong procedure or that my
hardware was faulty... but instead I did as you suggested and in the end.
everything about the procedure was perfect!!  you are a great service.
technical support!!  Thanks again

Il lun 11 set 2023, 19:31 Raphael Coeffic ***@***.***> ha
scritto:
